content/LGinC/drone-portainer/index.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2020-07-16T00:00:00+00:00
+title: "Portainer Stack Deploy"
+author: "LGinC"
+tags: [ deploy ]
+repo: "turinge/drone-portainer"
+logo: "portainer.svg"
+image: "turinge/drone-portainer"
+---
+
+The drone plugin can be used to deploy a Docker image to a Drone environment.The below pipeline configuration demonstrates simple usage:
+
+```yaml
+steps:
+- name: portainer
+  image: turinge/drone-portainer
+  settings:
+    serverurl: http://xxxxx:9000
+    username: 
+      from_secret: portainer_username
+    password:
+      from_secret: portainer_password
+    endpointId: 1
+    stackname: xxxservice
+    imagename: xxx/xxx
+    docker_compose: |
+      version: "2"
+      services:
+        xxx:
+          image: xxx/xxx
+          ports:
+          - 80:80
+```
+# Parameter Reference
+
+serverurl
+: portainer server url. like this: http://xxx.com:9000
+
+username
+: portainer username
+
+password
+: portainer password
+
+endpointId
+: portainer endpoint id,default 1, localhost is 1 <br> portainer终结点id，默认是1,即第一个，一般为localhost
```

Comment:
Please drop or translate these Chinese strings.

Reply:
first line is translation of Chinese strings

Reply:
But why the chinese translations? There is no chinese version of the index.

Reply:
can I create a Chinese_index.md? and add link in index.md

Reply:
The plugin index is English only, there is no Chinese version and it doesn't make sense to start that as none of the core maintainers is able to verify any translation beside the effort to maintain that at all.

Reply:
dropped it